Automattic/jetpack
# VideoPress: first step of the tracks control implementation

This PR is the first step to migrate the chapter's control to v6. Follow-up PRs coming on.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: first step of the tracks control implementation

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Block editor
* Add/edit a video block
* Ensure the block has some tracks
* Confirm you see the Tracks button in the Block controls.
* Open the dialog by clicking on the button
* Confirm it shows the Track list

No tracks | With tracks
------|------
<img width="317" alt="image" src="https://user-images.githubusercontent.com/77539/203636272-1e93a81a-ee98-4eae-8733-f8c8b57ccf33.png"> | <img width="318" alt="image" src="https://user-images.githubusercontent.com/77539/203636221-61a82f53-701c-4e21-96c1-33f24923e99e.png">
